### DISCUSSION TITLE
升级了管理器内核还未来得及升级空间就变2TB如何解决

### DISCUSSION THREAD

**Discussion by Do-Do666** [tiann/KernelSU#1388] [Q&A] [answered]
昨天晚上升级了管理器今天下午把内核升级最新版现在还是2TB是否需要等待一段时间？
（升级了管理器后刷了几个模块重启过就变2TB了![mmexport1708862026398.jpg](https://github.com/tiann/KernelSU/assets/161176501/916c4dd7-4b81-45c2-a2aa-fbd5f4aa35df)

）

**Comment by tiann** [comment, marked as answer]
https://kernelsu.org/zh_CN/guide/faq.html#%E4%B8%BA%E4%BB%80%E4%B9%88%E6%9C%89%E4%B8%AA-1t-%E7%9A%84%E8%B6%85%E5%A4%A7%E6%96%87%E4%BB%B6

**Reply by Do-Do666** [reply]
收到，感谢回复！

**Reply by Do-Do666** [reply]
我再问一下关闭模块现在加载半天是还在升级嘛
